Formula/goreman.rb

```diff
@@ -0,0 +1,42 @@
+require "language/go"
+
+class Goreman < Formula
+  desc "Foreman clone written in Go"
+  homepage "https://github.com/mattn/goreman"
+  url "https://github.com/mattn/goreman/archive/v0.1.1.tar.gz"
+  sha256 "d234ae81a397ab04338f26e30db79811715202413e0a3b9ff20395df7df675df"
+
+  depends_on "go"
+
+  go_resource "github.com/joho/godotenv" do
+    url "https://github.com/joho/godotenv.git",
+        :revision => "23d116af351c84513e1946b527c88823e476be13"
+  end
+
+  go_resource "github.com/daviddengcn/go-colortext" do
+    url "https://github.com/daviddengcn/go-colortext.git",
+        :revision => "186a3d44e9200d7eb331356ca4864f52708e1399"
+  end
+
+  go_resource "gopkg.in/yaml.v2" do
+    url "https://github.com/go-yaml/yaml.git",
+        :revision => "5420a8b6744d3b0345ab293f6fcba19c978f1183"
+  end
+
+  def install
+    gopath = buildpath/"gopath"
+    ENV["GOPATH"] = gopath
```

Comment:
```ruby
  def install
    ENV["GOPATH"] = buildpath
```

Reply:
I get a failure when I do this: 

$GOPATH/go.mod exists but should not

I think Go doesn't like it when you do the symlink like that, where it's alright to add things in a subdirectory.

Reply:
```ruby
  def install
    ENV["GOPATH"] = HOMEBREW_CACHE/"go_cache"
    (buildpath/"src/github.com/mattn").install buildpath.children

    cd "src/github.com/mattn" do
      system "go", "build", "-o", bin/"goreman"
      prefix.install_metafiles
    end
  end
```